recipes/pyslim/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pyslim" %}
+{% set version = "0.314" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 90fa4b9b15f5a009f9b9571b9d3fde85a184858ada631c242c4d379505dce39f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
This should usually be "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"

With `--no-deps`, you can then trim the list of `host` requirements to `python` and `pip` (or perhaps also `setuptools`)

Reply:
I see! I'll make the changes tomorrow. Thanks for the review.